src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -477,47 +471,6 @@ $facility-locator-shadow: rgba(0, 0, 0, 0.5);
   background-color: $color-red-dark;
 }
 
-#infoBubble {
```

Comment:
I couldn't find any DOM element with the `id` `infoBubble`. It was [added ~2.5 years ago](https://github.com/department-of-veterans-affairs/vets-website/pull/8651/files#diff-8dc9f9c094fb05328f951d321b0c6bd039a9dbbe990833d23a39b0a0a0326727R86) and got deleted somewhere along the way, but the CSS was left behind.

![image](https://user-images.githubusercontent.com/2008881/119412331-8fe6f800-bca0-11eb-8374-71215fe4dc3f.png)